pythia/tasks/vqa/visual_genome/builder.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import os
+import shutil
+
+from pythia.common.registry import registry
+from pythia.tasks.vqa.vqa2.builder import VQA2Builder
+from pythia.tasks.vqa.visual_genome.dataset import VisualGenomeDataset
+from pythia.utils.general import download_file, extract_file, get_pythia_root
+from pythia.common.constants import VISUAL_GENOME_CONSTS
+
+
+@registry.register_builder("visual_genome")
+class VisualGenomeBuilder(VQA2Builder):
+    def __init__(self):
+        super().__init__()
+        self.dataset_name = "visual_genome"
+        self.dataset_class = VisualGenomeDataset
+        self.writer = registry.get("writer")
+
+
+    def _build(self, dataset_type, config):
+        self._dataset_type = dataset_type
+        self._config = config
+        data_folder = os.path.join(get_pythia_root(), self._config.data_root_dir)
+
+        if self._dataset_type != "train":
+            return
```

Comment:
nit: a comment why only for `train` dataset we are downloading and extracting?

Reply:
IMDB link contains all of them to decrease the overall time. Thus, it is only downloaded for train.

Reply:
I will add a comment here.